components/local_ai/resources/local_ai.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Local AI Internals</title>
+    <style>
+      body {
+        font-family: system-ui, -apple-system, sans-serif;
+        margin: 20px;
+      }
+      iframe {
+        width: 100%;
+        height: 600px;
+        border: 1px solid #ccc;
+        margin-top: 20px;
+      }
+      .info {
+        background: #e3f2fd;
+        padding: 15px;
+        border-radius: 4px;
+        margin-bottom: 20px;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Local AI Internals</h1>
+    <div class="info">
+      <p><strong>Architecture:</strong></p>
+      <ul>
+        <li><strong>Trusted Page:</strong> chrome://local-ai-internals
+        (this page)</li>
+        <li><strong>Untrusted WASM:</strong>
```

Comment:
if this is supposed to be user-facing, should we explain here why it's untrusted?

Reply:
maybe we can add a label like `This feature is intended for developers; see [link to this PR] for more details.`

in case users find chrome://local-ai-internals and are confused what it's for

Reply:
added that in 6fee53aac5cb0058a6401ba8dfcd52c885b08ce7
<img width="805" height="235" alt="Screenshot 2025-11-20 at 15 56 15" src="https://github.com/user-attachments/assets/61b0e973-fa1e-4d54-a6d2-3f7379238e47" />